apollo-runtime/src/main/java/com/apollographql/apollo/ApolloCall.java

```diff
@@ -87,10 +87,11 @@
     public abstract void onFailure(@Nonnull ApolloException e);
 
     /**
-     * Gets called when the {@link #onResponse(Response)} has been called for the last time. Not called in the
-     * case of an error. {@link #onFailure(ApolloException)} will be called instead.
+     * Gets called whenever any action happen to this {@link ApolloCall}.
+     *
+     * @param event status event that corresponds to the action happen to this call
```

Comment:
Event status event sounds confusing could you reword a bit?

Reply:
Yeah, open to any suggestions

Reply:
Status that corresponds to a Apollo Call action

Reply:
oh, sorry I thought your comment was related to the name of method itself `onStatusEvent`.
:)